dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet50_wcServer/server.xml

```diff
@@ -0,0 +1,15 @@
+<server description="Server for testing Servlet 50 jakarta.servlet">
+
+  <include location="../fatTestPorts.xml"/>
+
+  <featureManager>
+    <feature>servlet-5.0</feature>
+  </featureManager>
+
+  <logging
+    traceSpecification="*=info=enabled:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all:HttpTransport=all:HTTPChannel=all:TCPChannel=all:GenericBNF=all:com.ibm.ws.jsp*=all:com.ibm.ws.session.*=all:com.ibm.ws.runtime.update.internal*=all:ChannelFramework=all"
+    maxFileSize="20"
+    maxFiles="10"
+    traceFormat="BASIC"/>
+
```

Comment:
Do we really need all the trace specified here , same comment for the bootstrap?